mystery_word.py

```diff
@@ -0,0 +1,206 @@
+# Jason Aylward
+# Homework 3
+import random
+import nightmare
```

Comment:
Kickass on the import here. Most people duplicated code between the two game versions. I really dig that you imported code from nightmare instead.